app/renderer/components/common/messageBox.js

```diff
@@ -154,7 +158,10 @@ const styles = StyleSheet.create({
     marginTop: globalStyles.spacing.dialogInsideMargin,
     minWidth: '425px',
     marginBottom: globalStyles.spacing.dialogInsideMargin,
-    userSelect: 'text'
+    userSelect: 'text',
+    maxHeight: '300px',
```

Comment:
that's great, I took a look at other browsers and they take that approach as well. Maybe 80vh is good enough?